### PR TITLE
Move governance documents to keptn/community

### PR DESCRIPTION
**Pull request by oleg-nenashev**
For https://github.com/keptn/keptn/issues/9091

### CODE REVIEW

**Review by thschue — APPROVED** [review]
lgtm, it makes sense to me to combine these things in the community repository

**Review by johannes-b — APPROVED** [review]
lgtm

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# Keptn Contributors
```
Comment (mowies) [inline CONTRIBUTORS.md:1]: Wouldn't it make sense to leave this file where it was? Contributions still come in on a repo level. Why move this file to a higher level?

Comment (AloisReitbauer) [reply CONTRIBUTORS.md:1]: I also think this will be hard to maintain.

File: COMMUNITY_MEMBERSHIP.md [hunk 2]
```diff
@@ -16,9 +16,9 @@ The Keptn maintainers own this document and process until delegated. They can be
...
-| Maintainer | Set direction and priorities for a subproject         | Sponsored by 2 maintainers. Demonstrates responsibility and excellent technical judgment for the whole project. | - [Maintainers of the Keptn GitHub organization](https://github.com/orgs/keptn/teams/maintainers) and [listed here](https://github.com/keptn/keptn/blob/master/MAINTAINERS)<br>- [CODEOWNERS](https://help.github.com/en/articles/about-code-owners) in [Keptn](https://github.com/keptn/keptn)<br> - Ownership of the GitHub repository |
+| Member     | Active contributor in the community and reviewer of PRs | Sponsored by 2 approvers or maintainers. Multiple (infrequent) contributions to the project. | [Members of the Keptn GitHub organization](https://github.com/orgs/keptn/teams/members) and [listed here](/MAINTAINERS.md)                          |
```
Comment (AloisReitbauer) [inline COMMUNITY_MEMBERSHIP.md:19]: There was never a GC or maintainer agreement about these roles.

**Review by AloisReitbauer — CHANGES_REQUESTED** [review]

**Comment by AloisReitbauer** [comment]
I would like to close this as there is no more activity on this issue and it has unresolved comments.

**Pull request closed without merging**